_episodes/l1-01-tools-I.md

```diff
@@ -174,51 +164,3 @@ to appear however.**
 
 If you already have a Python file open then it's also possible to set the
 interpreter using the toolbar at the bottom of the window.
-
-> ## Installing an editable package
```

Comment:
I think it is a good idea to get rid of this. It was not adding anything useful in an introductory course like this one.